railgun/config/variants/enterprise-postgres/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+resources:
+- ../postgres
+- enterprise-resources.yaml
+patchesStrategicMerge:
+- kong-enterprise.yaml
+images:
+- name: kong
+  newName: kong/kong-gateway
+  newTag: 2.4.1.0-alpine
```

Comment:
Similar to my above comment, since we're using such a specific version and security issues might get missed we should add a follow up Github issue to automate updates of this tag.